include/alpaka/acc/AccOmp5.hpp

```diff
@@ -58,36 +55,36 @@ namespace alpaka
             typename TIdx,
             typename TKernelFnObj,
             typename... TArgs>
-        class TaskKernelCpuOmp4;
+        class TaskKernelOmp5;
     }
     namespace acc
     {
         //#############################################################################
-        //! The CPU OpenMP 4.0 accelerator.
+        //! The CPU OpenMP 5.0 accelerator.
         //!
-        //! This accelerator allows parallel kernel execution on a CPU device.
-        //! It uses CPU OpenMP4 to implement the parallelism.
+        //! This accelerator allows parallel kernel execution on an OpenMP target device.
         template<
             typename TDim,
             typename TIdx>
-        class AccCpuOmp4 final :
+        class AccOmp5 final :
             public workdiv::WorkDivMembers<TDim, TIdx>,
-            public idx::gb::IdxGbRef<TDim, TIdx>,
+            public idx::gb::IdxGbLinear<TDim, TIdx>,
             public idx::bt::IdxBtOmp<TDim, TIdx>,
             public atomic::AtomicHierarchy<
-                atomic::AtomicStdLibLock<16>,   // grid atomics
+                atomic::AtomicOmpBuiltIn,   // grid atomics
```

Comment:
Is OpenMP guarantee that atomic is atomic between grids?
If not than it would not be allowed to use atomics in two parallel running kernel where the same memory is accessed. We need to at at least a warning about unexpected behaviour into the documentation.

Reply:
It depends:
```
If the size of x is 8, 16, 32, or 64 bits and x is aligned to a multiple of its size, the binding thread set
for the atomic region is all threads on the device. Otherwise, the binding thread set for the
atomic region is all threads in the contention group. atomic regions enforce exclusive access
with respect to other atomic regions that access the same storage location x among all threads in
the binding thread set without regard to the teams to which the threads belong.
```

Reply:
Did I understand the quoted fragment right that in case the size and alignment conditions are met, atomicity is guaranteed for all threads on a device (so between grids in alpaka speech), while if they are not met, only between threads of an openmp team?

Reply:
That is how I understood it.